src/commands/addImage.ts

```diff
@@ -59,8 +59,8 @@ export const builder = (yargs: Argv) => {
       type: "array",
       default: [],
     })
-    .option("import_data_image", {
-      describe: "The MD5 in which to inherit merkle data from",
+    .option("share_image_data_md5", {
```

Comment:
have to still use the import_data_image as option currently I need check with Xin about it but currently still use the not accurate name